packages/modal/README.md

```diff
@@ -4,32 +4,27 @@ Modals are streamlined, but flexible, dialog prompts with the minimum required f
 
 We use modals for creating, editing, and deleting objects as well as alert dialogs.
 
-## Modal sizes
+## Modal Sizes
 
-Designer/Developer can choose whether to use a large or small modal depending on the content.
+Opt for a large or small modal depending on the amount of content to display.
 
 ### Small Dialog Modal
 
-Small dialog modals are mostly used for alerts. E.g. confirm you want to continue with an action
+Small dialog modals are mostly used for alerts. For example, to confirm you want to continue with an action.
 
-Alerts can require some small interaction from the user, but should be kept as simple as possible. E.g.: requiring the user to type the name of a cluster to confirm it should be deleted.
+Alerts can require some small interaction from the user, but should be kept as simple as possible. For example, requiring the user to type the name of a cluster to confirm it should be deleted.
 
 ### Large Dialog Modal
 
-Large dialog modals are used when the user would benefit from seeing the content underlying the content of the modal.
-
-Large dialog modals are used for:
-
-- simple forms
-- content-heavy alerts
+Large dialog modals are used when the user would benefit from seeing the content underlying the content of the modal. These are generally used for content-heavy alerts. They can also be used for simple forms.
 
 ### Fullscreen Modal
 
 Fullscreen modals are used for complex interactions such as large forms. They let the user focus on the complexity and importance of the modal content by covering the entire viewport.
 
 Fullscreen modals are used for:
 
-- forms that affect high-priority features (e.g.: Kommander Projects, and Kommander Clusters)
+- forms that affect high-priority features (Kommander Projects, and Kommander Clusters)
 - forms that have many input fields
 - content that requires navigation tabs
```

Comment:
Should we capitalize these bullet points? All the bullets in the other docs seem to be.

Reply:
yep definitely!